### ISSUE TITLE
Undefined variable when using symbols

### ISSUE THREAD

**Issue by Datseris** [JuliaDocs/Documenter.jl#612]
This is an example I used:

````markdown
```@example 1
using DynamicalSystems
using PyPlot
PyPlot.ioff() # hide

ds = Systems.logistic()
i = 1
pvalues = 2:0.001:4
ics = [rand() for m in 1:10]
n = 50
Ttr = 5000
parameter = :r 
output = orbitdiagram(ds, i, parameter, pvalues; n = n, Ttr = Ttr)

figure()
for (j, p) in enumerate(pvalues)
    plot(p .* ones(output[j]), output[j], lw = 0,
    marker = "o", ms = 0.5, color = "black")
end
xlabel("\$r\$"); ylabel("\$x\$")
savefig("logisticdiagram.png"); nothing # hide
```
````

The line `parameter = :r` gives an `UndefVarError [:r]` when I run make.jl . If I change this to not defining `parameter` and instead passing the value immediately to the function, I wasn't getting the error anymore:
`output = orbitdiagram(ds, i, :r, pvalues; n = n, Ttr = Ttr)`

**Comment by mortenpi** [comment]
Can reproduce. My guess is that Documenter gets confused with the RHS, thinking the `Symbol` literal is a variable symbol.

**Comment by mortenpi** [comment]
Fixed by #614.
